lib/commands.py

```diff
@@ -65,8 +65,8 @@ def _ellide_log_lines(log):
 OUTPUT_HANDLER.setFormatter(logging.Formatter('%(message)s'))
 
 def _ssh(hostname_or_ip, cmd, check=True, simple_output=True, suppress_fingerprint_warnings=True,
-         background=False, target_os='linux', decode=True):
-    options = []
```

Comment:
unrelated to the PR?

Reply:
It's in a separate commit that I will move first before merging, but that I left at the end to not make the review harder. The options new parameter is necessary. The timeout thing is also useful to avoid having the script hang if you use the wrong host IP.